### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by Jordie0608**
See branch name.

### CODE REVIEW

File: html/changelog.html [hunk 1]
```diff
@@ -55,6 +55,46 @@
...
+				<li class="rscadd">Nanotrasen scientists have recently reported mutations within the pink family line of slimes. Station scientists are encouraged to investigate.</li>
+			</ul>
+			<h3 class="author">Paprika updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="tweak">Reworked reskinning and renaming guns, and added the ability to reskin the bartender's shotgun. Click it with an empty active hand to reskin. Warning, you can only do this once! Rename by hitting it with a pen. This also works with miner shotguns now.</li>
+			</ul>
+			<h3 class="author">RemieRichards updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="wip">Embedding from explosions has been tweaked, they will always throw objects fast enough to get embedded if they can.</li>
+				<li class="tweak">Damage taken when embedded objects fall out has been increased.</li>
```
Comment (RemieRichards) [inline html/changelog.html]: but this is not true

Comment (Jordie0608) [inline html/changelog.html]: Maybe you should've made the changelog yml yourself then =P

Comment (RemieRichards) [inline html/changelog.html]: How did I know you'd make that comeback...

Comment (Jordie0608) [inline html/changelog.html]: If you pass me a corrected entry I'll update it.

Comment (RemieRichards) [inline html/changelog.html]: Can't, I'm not at my home computer.
plus I'm lazy so :P

Comment (Jordie0608) [inline html/changelog.html]: I meant like type here what it should say.

Comment (RemieRichards) [inline html/changelog.html]: Oh well, I've already done that technicallu. that line in the entry is false, nothing changed so it really shouldn't be in the changelog.

Unless you want to mention *some* embedding weapons now have unique damage values.

**Pull request merged**